ingester2/src/server/grpc/query.rs

```diff
@@ -205,10 +201,9 @@ where
             )
             .await?;
 
-        let output = FlightFrameCodec::new(
-            FlatIngesterQueryResponseStream::from(response),
-            self.ingester_id,
-        );
+        let output = encode_response(response, self.ingester_id)
+            // map FlightError --> tonic::Status
+            .map_err(|e| e.into());
```

Comment:
`Into::into()` conversions are often fairly opaque when it comes to the types, comments like this help (thanks! :heart:) but we can also invert the conversion and have the description in code & type-checked:

```suggestion
        let output = encode_response(response, self.ingester_id).map_err(tonic::Status::from);
```

This matches the other conversions in this file too :+1:

Reply:
Good idea -- will do